package.json

```diff
@@ -24,12 +24,13 @@
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -p tslint-rules",
     "compile:script": "tsc -p script",
-    "lint": "yarn compile:tslint && yarn compile:script && yarn tslint && yarn eslint-check && yarn eslint",
-    "lint:fix": "yarn compile:tslint && yarn tslint:fix && yarn eslint:fix",
+    "lint": "yarn lint:src && yarn lint:prettier || (echo '\u001b[1m\u001b[32m→ To fix some of these errors, run \u001b[39m\u001b[22m\u001b[1m\u001b[32m\u001b[4myarn lint:fix\u001b[24m\u001b[39m\u001b[22m' && false)",
```

Comment:
The use of `false` here doesn't work for non-bash shells. This affects both CMD and PowerShell on Windows:

```shellsession
> yarn lint
yarn run v1.3.2
$ yarn lint:src && yarn lint:prettier || (echo '→ To fix some of these errors, run yarn lint:fix' && false)
$ yarn tslint && yarn eslint-check && yarn eslint
$ tslint ./{script,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}
$ eslint --print-config .eslintrc.* | eslint-config-prettier-check
No rules that are unnecessary or conflict with Prettier were found.
$ ts-node script/eslint.ts

$ prettier **/*.md **/*.scss --list-different
.github\ISSUE_TEMPLATE.md
docs\contributing\linux-testing-and-you.md
docs\technical\syntax-highlighting.md
app\styles\ui\_repository-list.scss
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
'→ To fix some of these errors, run yarn lint:fix'
'false' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I think we might have to use the `prettier` API here if we want better control of the output and exit codes across multiple shells. I'll take some time today to see if I can PR something into your branch that's similar to `script/eslint.ts`.

Reply:
See https://github.com/j-f1/forked-desktop/pull/1 for a proposed solution